src/platform/packages/shared/kbn-rule-data-utils/src/default_alerts_as_data.ts

```diff
@@ -145,6 +145,7 @@ const ALERT_SCHEDULED_ACTION_THROTTLING = `${ALERT_SCHEDULED_ACTION_NAMESPACE}.t
 
 // kibana.alert.index_pattern - index pattern for the alert
 const ALERT_INDEX_PATTERN = `${ALERT_NAMESPACE}.index_pattern` as const;
+const ALERT_ATTACK_IDS = `${ALERT_NAMESPACE}.attack_ids` as const;
```

Comment:
Since we decided to move this field to security and attack discovery alerts, I think it might be better to define this variable outside the `kbn-rule-data-utils` in more relevant context.

I was thinking that `x-pack/platform/packages/shared/kbn-elastic-assistant-common/constants.ts` could be a good place for it. We could then import this variable from both `elastic_assistant` and `security_solution` plugins.

Wdyt @NicholasPeretti?